Dockerfile

```diff
@@ -8,12 +8,27 @@ RUN rm -r /context/etc
 
 
 FROM base_image AS base_image_python
+
 RUN apt-get update && \
     apt-get full-upgrade -y --no-install-recommends && \
     apt-get install -y --no-install-recommends python3-pip && \
     python3 -m pip install --no-cache-dir -U pip setuptools wheel
-#RUN apt-get install -y --no-install-recommends wget libcap2-bin
 
+RUN apt-get install -y --no-install-recommends wget libcap2-bin
+
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG PATH="/root/miniconda3/bin:${PATH}"
+RUN wget \
+    https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
```

Comment:
You should really hardcode a version number, cause some day it'll break and it'll be impossible to debug.

Reply:
see https://docs.conda.io/en/latest/miniconda.html#linux-installers